frontend/src/components/PhotoOverlay.css

```
@@ -0,0 +1,28 @@
+      to top,
+      rgba(0, 0, 0, 0.82) 0%,
+      rgba(0, 0, 0, 0.55) 60%,
+      rgba(0, 0, 0, 0) 100%
+   );
+   color: #fff;
+   font-size: 0.95rem;
+   font-weight: 700;
+   line-height: 1.2;
+   text-align: left;
+   pointer-events: none;
+   word-wrap: break-word;
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)